### PR TITLE
add "join table" for visits

### PR DESCRIPTION
**Pull request by GhostEsso**
Hello **Code Reviewer**,
I hope you're doing well :sunglasses:
In this Milestone:
- I created a table named vets;
- I inserted the data for vets;
- I inserted the data for specializations;
- I inserted the data for visits;

I wrote **queries** to answer the questions(screenshots below):

[Who was the last animal seen by William Tatcher?
](https://github.com/GhostEsso/Vet-Clinic/assets/114746721/86796c9d-c4d5-4f8f-9f4a-79e597b7d0df)
[How many different animals did Stephanie Mendez see?
](https://github.com/GhostEsso/Vet-Clinic/assets/114746721/c4372fb0-bd76-4b9e-a824-f8f5254bca88)
[List all vets and their specialties, including vets with no specialties.
](https://github.com/GhostEsso/Vet-Clinic/assets/114746721/f0d77909-0569-4e89-a71c-4c754d79043f)
[List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
](https://github.com/GhostEsso/Vet-Clinic/assets/114746721/fd2704cb-3077-4316-a897-5bdf02fc1b45)
[What animal has the most visits to vets?
](https://github.com/GhostEsso/Vet-Clinic/assets/114746721/eac793da-4b16-45a4-b70d-7b09eb4b4fae)
[Who was Maisy Smith's first visit?
](https://github.com/GhostEsso/Vet-Clinic/assets/114746721/b03a00dd-a9fb-41c8-ba5c-df64d2d1d7c8)
[Details for most recent visit: animal information, vet information, and date of visit.
](https://github.com/GhostEsso/Vet-Clinic/assets/114746721/e9d7911d-7198-4168-9746-b6e838a17363)
[How many visits were with a vet that did not specialize in that animal's species?
](https://github.com/GhostEsso/Vet-Clinic/assets/114746721/cd82d925-8e94-4c83-83b9-872d848790b7)
[What specialty should Maisy Smith consider getting? Look for the species she gets the most.
](https://github.com/GhostEsso/Vet-Clinic/assets/114746721/57b596ab-a6fb-4697-80f0-bc215673b3d9)

Thanks for reviewing my Pull Request :pray:

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -38,3 +38,67 @@ CREATE TABLE species (
...
+  (7, 3, '2019-09-29'), -- Squirtle visited Stephanie Mendez on Sep 29th, 2019
+  (8, 4, '2020-10-03'), -- Angemon visited Jack Harkness on Oct 3rd, 2020
+  (8, 4, '2020-11-04'), -- Angemon visited Jack Harkness on Nov 4th, 2020
+  (9, 2, '2019-01-24'), -- Boarmon visited Maisy Smith on Jan 24th, 2019
+  (9, 2, '2019-05-15'), -- Boarmon visited Maisy Smith on Mai 15th, 2019
+  (9, 2, '2020-02-27'), -- BVMSO Feb 27th, 2020
+  (9, 2, '2020-08-03'), -- Boarmon visited Maisy Smith on Feb 27th, 2020
+  (10, 3, '2020-05-24'), -- Blossom visited SM on May 24th, 2020
+  (10, 1, '2021-01-11'); -- BlVWTO Jan 11th, 2021
+
```
Comment (Strangeal) [inline schema.sql]: - [ ] Great job with your database queries! 👏 However, I'd like to suggest a small improvement. To maintain a well-organized codebase and adhere to best practices, it's a good idea to save all queries related to populating tables with data in a dedicated `data.sql` file. This not only keeps your code tidy but also makes it easier for others (or your future self) to find and manage these queries. Happy coding! 😊🚀🍀

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @GhostEsso, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Well structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (GhostEsso) [reply schema.sql]: @Strangeal Thanks :pray: Understood CR, I'll do that.

**Review by Bantealem — APPROVED** [review]
# Status: Approved

Hi @GhostEsso, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [x] Correct Git flow
- [x] Descriptive PR summary and title
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**